sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
> What I do not understand is why; the docs clearly state that these symbols should be in, in this particular package.

It doesn't mean that you should export these symbols in the global namespace.  Is that clear?